### PR TITLE
feat: store/extend

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* divide extend func
* add store test

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: tests/store/stackSeriesData.spec.ts [hunk 1]
```diff
@@ -1,116 +1,84 @@
-import seriesData from '@src/store/seriesData';
+// import seriesData from '@src/store/seriesData';
```
Comment (jungeun-cho) [inline tests/store/stackSeriesData.spec.ts:1]: 지워도될것같아요.

File: tests/store/stackSeriesData.spec.ts [hunk 2]
```diff
@@ -1,116 +1,84 @@
...
-        options: {
-          series: {
-            stack: true
-          }
-        },
-        series: {
-          bar: data
-        }
-      });
+  it('should be initialized with default values, if the stack option is true only', () => {
```
Comment (jungeun-cho) [inline tests/store/stackSeriesData.spec.ts:14]: 이쪽은 initialize 테스트라
`describe('initialize')`로 묶고, 

아래 `describe('StackData', ...`는
`describe('setStackSeriesData'...`로 액션 테스트 하도록 
describe만 변경하면 좋을것같습니다.(부탁합니다.ㅎㅎㅎ)

File: tests/store/stackSeriesData.spec.ts [hunk 3]
```diff
@@ -1,116 +1,84 @@
...
-        }
-      });
+    expect(state.stackSeries.bar!.stack).toEqual({
+      type: 'percent',
+      connector: { type: 'dashed', color: '#ff0000', width: 1 }
     });
   });
 
   describe('StackData', () => {
     it('makeStackData', () => {
```
Comment (shiren) [inline tests/store/stackSeriesData.spec.ts:60]: 무슨 함수인줄 ㅋㅋ

@jungeun-cho TC 디스크립션을  it을 활용해서 문장으로 만들어줘~ 이거 니가 작성한거 맞지??
이 PR 머지 된후 작업 부탁

it('should make stack 블라불라

Comment (jungeun-cho) [reply tests/store/stackSeriesData.spec.ts:60]: 넵! 확인했습니다.

@jung-han 이쪽 TC파일은 전체적으로 제가 수정할께요 ㅎㅎ

**Pull request merged**